### DISCUSSION TITLE
不能获得小爱音箱pro的语音识别结果

### DISCUSSION THREAD

**Discussion by wangdongfreesky** [XiaoMi/ha_xiaomi_home#382] [Ideas]
不能获得小爱音箱pro的语音识别结果，xiaomi miot auto可以通过小爱音箱Pro 播放控制 conversation，来获得小爱同学识别出来的语音文字，进而通过homeassistant来判断运行其他命令

**Comment by JimmyKmi** [comment]
同
https://github.com/XiaoMi/ha_xiaomi_home/issues/366
https://github.com/XiaoMi/ha_xiaomi_home/discussions/283
https://github.com/XiaoMi/ha_xiaomi_home/discussions/257
